libraries/timeline-view/src/main/java/com/trendyol/timelineview/Extensions.kt

```diff
@@ -0,0 +1,23 @@
+package com.trendyol.timelineview
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+
+internal fun <T : ViewDataBinding> ViewGroup?.inflate(
+    @LayoutRes layoutId: Int,
+    attachToParent: Boolean = true
+): T {
+    if (this?.isInEditMode == true) {
+        View.inflate(context, layoutId, parent as? ViewGroup?)
+    }
+    return DataBindingUtil.inflate(
```

Comment:
This doesn't fix the issue about databinding in layout preview since it calls DataBindingUtils.inflate even view is in edit mode.

Reply:
fix as use edit mode control to view